contrib/build_sysimg.jl

```diff
@@ -162,8 +162,13 @@ function link_sysimg(sysimg_path=nothing, cc=find_system_compiler(), debug=false
     end
 
     info("Linking sys.$(Libdl.dlext)")
-    run(`$cc $FLAGS -o $sysimg_path.$(Libdl.dlext) $sysimg_path.o`)
-
+    if is_windows()
+        mv("$sysimg_path.$(Libdl.dlext)","$sysimg_path.$(Libdl.dlext).old";remove_destination=true)
+        run(`$cc $FLAGS -o $sysimg_path.$(Libdl.dlext) $sysimg_path.o`)
+        run(`rm "$sysimg_path.$(Libdl.dlext).old"`)
```

Comment:
Note `rm(....)` will not work  errors with
```julia
ERROR: unlink: operation not permitted (EPERM)
 in uv_error at .\libuv.jl:68 [inlined]
 in unlink(::String) at .\file.jl:382
 in #rm#7(::Bool, ::Bool, ::Function, ::String) at .\file.jl:107
 in link_sysimg(::String, ::String, ::Bool) at C:\Julia\julia-0.5-latest\share\julia\build_sysimg.jl:168
 in (::##2#3{Bool,String})() at C:\Julia\julia-0.5-latest\share\julia\build_sysimg.jl:90
 in cd(::##2#3{Bool,String}, ::String) at .\file.jl:48
 in #build_sysimg#1(::Bool, ::Bool, ::Function, ::Void, ::String, ::Void) at C:\Julia\julia-0.5-latest\share\julia\build_sysimg.jl:48
 in (::#kw##build_sysimg)(::Array{Any,1}, ::#build_sysimg, ::Void, ::String, ::Void) at .\<missing>:0 (repeats 2 times)
```

Another option is to just remove  run(`rm ....`) and leave thing's so that   sys.dll.old always refers to the last built image, although I prefer to have it deleted.